package-system/pybind11/Findpybind11.cmake

```
@@ -18,7 +18,9 @@ add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
 ly_target_include_system_directories(TARGET ${TARGET_WITH_NAMESPACE} INTERFACE ${${LIB_NAME}_INCLUDE_DIR})
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    target_compile_options(3rdParty::pybind11 INTERFACE -fsized-deallocation)
+    if(NOT MSVC)
+        target_compile_options(3rdParty::pybind11 INTERFACE -fsized-deallocation)
+    endif()
 endif()
```

Maybe we should enable the associated flag on MSVC?

```suggestion
if (MSVC)
    target_compile_options(3rdParty::pybind11 INTERFACE /Zc:sizedDealloc)
else if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
    target_compile_options(3rdParty::pybind11 INTERFACE -fsized-deallocation)
endif()
```

---

done